src/main/java/com/tenx/ms/retail/config/RetailResourceServerConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.tenx.ms.retail.config;
```

Comment:
This ResouceServerConfiguration class is not needed in this project as you don't have security service to back it up and also don't have any controller wired up with security.